CodeMaid/Logic/Cleaning/FileHeaderLogic.cs

```diff
@@ -68,6 +73,58 @@ internal void UpdateFileHeader(TextDocument textDocument)
             }
         }
 
+        /// <summary>
+        /// Replace the variables in the header if any
+        /// </summary>
+        /// <param name="textDocument"></param>
+        internal void ReplaceVariables(TextDocument textDocument, ref string settingsFileHeader)
+        {
+            foreach (var variable in _variables)
```

Comment:
This approach will be making calls out to the IDE every time it is invoked.. even if the header doesn't contain any of these variables.

I think perhaps instead it would make sense to scan the header for the variables, determine which were present, calculate the replacement and put it in place.